examples/dreambooth/train_dreambooth.py

```diff
@@ -107,6 +108,12 @@ def save_model_card(
         f.write(yaml + model_card)
 
 
+def unwrap_model(accelerator, model):
```

Comment:
Let's make this inside the function as done in the other PR.

Reply:
`log_validation` lies outside of the function. Should I put `log_validation` inside too?

Reply:
`log_validation` is fine since it's a big one. However, `unwrap_model()` isn't.

Reply:
`log_validation` uses `unwrap_model` too so moving `unwrap_model` inside the function will raise error. Do you have any idea how to fix this?

Reply:
We can pass the model to `log_validation` in the unwrapped state, no?

Reply:
I see. I have made new update